### ISSUE TITLE
Emails from app.phpmentoring.org are marked as Spam in Gmail

### ISSUE THREAD

**Issue by Djuki** [phpmentoring/testapp#67]
Hi,

I just want to let you know that since November emails from http://app.phpmentoring.org/ ended up in my spam folder.

Before that i received emails from phpmentoring in my inbox. For some reason Gmail filter emails as spam. This is why I did not notice that My inbox had unread messages.

You can count unread messages as (one of) indicator how many people did not get notification emails.

**Comment by tdutrion** [comment]
I reported the same issue #46 but at the time it seemed to have been fixed.

I also experienced the problem today, and here his the interesting part of the original message:

```
Received-SPF: neutral (google.com: 104.236.23.66 is neither permitted nor denied by best guess record for domain of no-reply@phpmentoring.org) client-ip=104.236.23.66;
Authentication-Results: mx.google.com;
       spf=neutral (google.com: 104.236.23.66 is neither permitted nor denied by best guess record for domain of no-reply@phpmentoring.org) smtp.mailfrom=no-reply@phpmentoring.org
```

Looks like the SPF record has not been set according to [mxtoolbox](http://mxtoolbox.com/domain/app.phpmentoring.org/), maybe it has not been set when transitioning to Cloudflare.

**Comment by auroraeosrose** [comment]
This should be fixed when we move the application itself to a third party email provider (see #74)

**Comment by svpernova09** [comment]
resolved via #74
